srml/support/test/tests/final_keys.rs

```diff
@@ -14,87 +14,211 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-use runtime_io::with_externalities;
-use primitives::Blake2Hasher;
+use runtime_io::with_storage;
 use support::storage::unhashed;
-use codec::{Encode, Decode};
+use codec::Encode;
+use support::{StorageDoubleMap, StorageLinkedMap, StorageMap, StorageValue};
 
-pub trait Trait {
-	type Origin;
-	type BlockNumber: Encode + Decode + Default + Clone;
-}
+mod no_instance {
+	use codec::{Encode, Decode};
```

Comment:
The indentation here is a bit weird, should just be a regular tab?

Reply:
I don't get it, this is a regular tab

Reply:
oh I think it is related to github way to render it. maybe they count the "+ " into the tab here and this end into this. weird indeed, but in the code this is a tab

Reply:
Ah, fair enough :)